src/olympia/devhub/templates/devhub/addons/submit/describe.html

```diff
@@ -40,6 +40,27 @@ <h3>{{ _('Describe Add-on') }}</h3>
              data-maxlength="{{ form.summary.field.max_length }}"></div>
       </div>
     </div>
+    {% if addon.type == amo.ADDON_EXTENSION %}
+    <div class="addon-submission-field">
+      <label>{{ _('Description:') }}
+      {% if form.description.field.min_length %}
+        <span class="req" title="{{ _('required') }}">*</span>
+      {% endif %}</label>
+      {{ form.description }}
+      {{ form.description.errors }}
+      <div class="edit-addon-details">
+        {{ _('A longer explanation of features, functionality, and other relevant information. '
+             "This field is only displayed on the add-on\'s listing. " +
```

Comment:
IF you're using quotes you don't have to escape the single-quote. But let's use regular quotes for the whole string.